apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -23,8 +23,8 @@ defmodule Aecore.Chain.Worker do
     genesis_block_map = %{genesis_block_hash => Block.genesis_block()}
     genesis_chain_state = ChainState.calculate_block_state(Block.genesis_block().txs)
     latest_block_chain_state = %{genesis_block_hash => genesis_chain_state}
-
-    initial_state = {genesis_block_map, latest_block_chain_state}
+    txs_index = %{}
```

Comment:
build the initial index from the `Block.genesis_block().txs` even if there are none